<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopDrinkReportGui</class>
 <widget class="QDialog" name="TopDrinkReportGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Top Drink Report</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="6">
          <widget class="QDateEdit" name="endDateEdit">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="dateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2022</year>
             <month>12</month>
             <day>20</day>
            </datetime>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="alcoholTypeSelection">
           <property name="toolTip">
            <string>Type of alcohol to select from</string>
           </property>
           <property name="currentText">
            <string>All Types</string>
           </property>
           <property name="maxVisibleItems">
            <number>9</number>
           </property>
           <item>
            <property name="text">
             <string>All Types</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Beer</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Liquor</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Wine</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>and</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Select top</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="topNumberSelection">
           <property name="toolTip">
            <string>Number of records to select</string>
           </property>
           <property name="minimum">
            <number>3</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QDateEdit" name="startDateEdit">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="dateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2022</year>
             <month>12</month>
             <day>20</day>
            </datetime>
           </property>
           <property name="displayFormat">
            <string>M/d/yyyy</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>where rating &gt;=</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>between</string>
           </property>
          </widget>
         </item>
         <item row="0" column="8">
          <widget class="QSpinBox" name="ratingSelection">
           <property name="maximum">
            <number>10</number>
           </property>
           <property name="value">
            <number>7</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QTableWidget" name="queryResults">
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Subtype</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Producer</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Rating</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
